docker/everest-playground-opensuse-tumbleweed/.devcontainer.json

```diff
@@ -21,7 +21,12 @@
   "workspaceFolder": "/workspace/everest-cpp",
```

Comment:
I would prefer something more meaningful than everest-cpp.  We should also discuss the usage of vscode devcontainer files